### PR TITLE
feat: add createSerializer

### PR DESCRIPTION
**Pull request by eunjae-lee**
## Summary

This PR adds `createSerializer` which is a factory that accepts options for diffable-html.

By using this, users can pass `sortAttributes` option to `diffable-html`.

```js
import createSerializer from 'jest-serializer-html/createSerializer';

createSerializer({
  print: {
    sortAttributes: (names) => names.sort()
  }
});
```

### CODE REVIEW

File: asc.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+var toDiffableHtml = require('@algolia/diffable-html');
+
+module.exports = {
+  test(object) {
```
Comment (Haroenv) [inline asc.js:4]: the content of the three functions is almost the same, could you export the `test` from somewhere common?

**Review by Haroenv — COMMENTED** [review]
too bad options can't be passed to pretty-format plugins

Alternatively to this PR, we could export a plugin factory and pass objects before passing it to addSnapshotSerializer in the jest setup

**Comment by eunjae-lee** [comment]
> too bad options can't be passed to pretty-format plugins
> 
> Alternatively to this PR, we could export a plugin factory and pass objects before passing it to addSnapshotSerializer in the jest setup

```js
// scripts/custom-jest-serializer-html.js

import createSerializer from 'jest-serializer-html/createSerializer';

module.exports = createSerializer({
  print: {
    sortAttributes: (names) => names.sort()
  }
});
```

```js
// package.json

"jest": {
  "snapshotSerializers": [
    "./scripts/custom-jest-serializer-html"
  ]
}
```

What do you think?

**Comment by Haroenv** [comment]
that could work, or even calling addSnapshotSerializer in the jest-setup file

**Review by Haroenv — APPROVED** [review]
not fully sure about the name `print`, but not all that big of a deal

**Pull request merged**